docs/host_install.md

```diff
@@ -76,28 +76,28 @@ echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic
 wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
 apt-get update
 apt-get build-essential python #dependencies
-apt-get install -y libsgx-enclave-common sgx-aesm-service libsgx-urts libsgx-launch libsgx-epid libsgx-quote-ex libsgx-uae-service
+apt-get install -y sgx-aesm-service libsgx-urts libsgx-uae-service
 ```
 
-if you want to debug and/or develop, also install following packages
+if you want to debug, also install following packages
 ```bash
-apt-get install -y libsgx-enclave-common-dbgsym libsgx-enclave-common-dev
+apt-get install -y libsgx-enclave-common-dbgsym sgx-aesm-service-dbgsym libsgx-urts-dbgsym libsgx-uae-service-dbgsym
 ```
 
 ## Install the SGX SDK
 
-Private Data Objects has been tested with version 2.9.1 of the SGX
+Private Data Objects has been tested with version 2.10 of the SGX
 SDK. You can download prebuilt binaries for the SDK and kernel drivers
-from [01.org](https://download.01.org/intel-sgx/sgx-linux/2.9.1/distro/ubuntu18.04-server/).
+from [01.org](https://download.01.org/intel-sgx/sgx-linux/2.10/distro/ubuntu18.04-server/).
 
-The following commands will download and install version 2.9.1 of the SGX
+The following commands will download and install version 2.10 of the SGX
 SDK. When asked for the installation directory, we suggest that you install
 the SDK into the directory `/opt/intel`.
 
 ```bash
-UBUNTU_VERSION=ubuntu18.04-server
-DRIVER_REPO= https://download.01.org/intel-sgx/sgx-linux/2.9.1/distro/${UBUNTU_VERSION}/
-SDK_FILE=sgx_linux_x64_sdk_2.9.101.2.bin
+UBUNTU_VERSION=ubuntu18.04
+DRIVER_REPO=https://download.01.org/intel-sgx/sgx-linux/2.10/distro/${UBUNTU_VERSION}-server/
+SDK_FILE=sgx_linux_x64_sdk_2.10.100.2.bin
 
```

Comment:
by the way... we have lost the ability to cut & paste the commands into a shell. if you are writing the instructions as a script then you should probably just put them into a script and include it. otherwise... add all the exports for the variables and the sudo to install

Reply:
why are saying we can't do copy/paste?  That's how i tested, again yesterday trying to replicate marcela's problems.

Reply:
i have to admit i've done a `sudo -i` before copy/pasting ...

Reply:
i'll add the missing sudo (not that i stripped them ;-)